src/openfermion/utils/_operator_utils_test.py

```diff
@@ -333,12 +333,27 @@ def test_save_and_load_fermion_operators(self):
                          msg=str(self.fermion_operator -
                                  loaded_fermion_operator))
 
+    def test_save_and_load_fermion_operators_readably(self):
+        save_operator(self.fermion_operator, self.file_name,
+                      human_readable=True)
+        loaded_fermion_operator = load_operator(self.file_name,
+                                                human_readable=True)
+        self.assertEqual(self.fermion_operator.terms,
+                         loaded_fermion_operator.terms)
```

Comment:
`self.assertEqual(self.fermion_operator, loaded_fermion_operator)`

Reply:
Should the rest of the test cases be comparing the original operator and the loaded operator in this way? Including the cases that have already been written?

Reply:
That would be nice but I won't require it. Those tests were actually written before this was possible; we didn't always implement `__eq__` in SymbolicOperator.